src-d/rules_python/python/runfiles/README.rst

```diff
@@ -0,0 +1,56 @@
+bazel-runfiles library
+======================
+
+This is a Bazel Runfiles lookup library for Bazel-built Python binaries and tests.
```

Comment:
This is likely stale as `main` has an `md` file: https://github.com/bazelbuild/rules_python/blob/main/python/runfiles/README.md

Reply:
Also ensure that examples use `rules_python`